composer.json

```diff
@@ -15,7 +15,7 @@
     "scripts": {
         "cs:fix": "php-cs-fixer fix",
         "cs:check": "php-cs-fixer fix --dry-run --diff",
-        "lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
+        "lint": "find . -name \\*.php -not -path './vendor/*' -not -path './tests/*' -print0 | xargs -0 -n1 php -l",
```

Comment:
@ChristophWurst Is it really a good idea to exclude the tests from linting completely? They don't get linted at all now, not only not linted with PHP 7.2.

Reply:
I guess they will break anyway when executing them, so it doesn't matter much.

Reply:
Well, one could fix them for 7.2 as well 😉

Reply:
php7.2 is EOL. we will make this work for 3.0.x but it will be dropped for 3.1

Reply:
> we will make this work for 3.0.x but it will be dropped for 3.1

Then maybe only merge this into the stable3.0 branch?

Reply:
we'll merge, backport and then drop nextcloud 20 and php7.2